### PR TITLE
small fix of example, added validator blocks found

### PR DESCRIPTION
**Pull request by bwhm**
More stuff to be added...

### CODE REVIEW

File: CONTRIBUTIONS.md [hunk 1]
```diff
@@ -14,10 +14,17 @@ As we learn more about the evolving ways in which our community can contribute,
...
   - because there is little additional benefit in terms of learning about the project from setting up more than one node, we are likely to offer smaller rewards for further nodes you set up, though these will still qualify for rewards.
   - you should also be aware that validators can earn further formal rewards on our current testnet
-
+  - note that you can only get points for a single validator, as these are also rewarded in tokens
+- running a node [indicative score: up to 50 per day]
+  - running a node (that is not validating) can have almost the same value as running a validator, is cheaper, easier, and comes with less risk:
+    - you do not have to to run your node with `--pruning="archive"` (less storage)
+    - you don't need as good hardware, or connection (to other vals), to quickly validate and produce blocks
+    - you don't need to stake, thus no risk getting slashed if your node goes down
+    - on the downside, you do not produce blocks for the network, and don't earn block rewards
```
Comment (blrhc) [inline CONTRIBUTIONS.md:26]:
```suggestion
    - on the downside, you do not produce blocks for the network, and don't earn formal block rewards
```

File: CONTRIBUTIONS.md [hunk 2]
```diff
@@ -14,10 +14,17 @@ As we learn more about the evolving ways in which our community can contribute,
 
 Network Integrity:
 - running a validator [indicative score: 50 per day]
-  - since running even a validator node contributes to the security of the platform and allows you to hook yourself up to the heart of our network, we want to recognize and reward this common and relatively straightforward first step towards further engagement in the project.
+  - since running a validator node contributes to the security of the platform and allows you to hook yourself up to the heart of our network, we want to recognize and reward this common and relatively straightforward first step towards further engagement in the project.
   - because there is little additional benefit in terms of learning about the project from setting up more than one node, we are likely to offer smaller rewards for further nodes you set up, though these will still qualify for rewards.
   - you should also be aware that validators can earn further formal rewards on our current testnet
-
+  - note that you can only get points for a single validator, as these are also rewarded in tokens
```
Comment (blrhc) [inline CONTRIBUTIONS.md:20]:
```suggestion
  - note that you can only get leaderboard points for a single validator, as these are also rewarded in testnet tokens
```

File: CONTRIBUTIONS.md [hunk 3]
```diff
@@ -14,10 +14,17 @@ As we learn more about the evolving ways in which our community can contribute,
 
 Network Integrity:
 - running a validator [indicative score: 50 per day]
-  - since running even a validator node contributes to the security of the platform and allows you to hook yourself up to the heart of our network, we want to recognize and reward this common and relatively straightforward first step towards further engagement in the project.
+  - since running a validator node contributes to the security of the platform and allows you to hook yourself up to the heart of our network, we want to recognize and reward this common and relatively straightforward first step towards further engagement in the project.
   - because there is little additional benefit in terms of learning about the project from setting up more than one node, we are likely to offer smaller rewards for further nodes you set up, though these will still qualify for rewards.
```
Comment (blrhc) [inline CONTRIBUTIONS.md:18]:
```suggestion

```

File: CONTRIBUTIONS.md (hunk 1 above)
Comment (bwhm) [reply CONTRIBUTIONS.md:26]: I'll leave this one out, as there are not "informal" block rewards :)

Comment (bwhm) [reply CONTRIBUTIONS.md:26]: Waiting for you to finish before I reply to the rest!

File: README.md [hunk 4]
```diff
@@ -66,6 +67,17 @@ Some important considerations:
 
 `*` Jsgenesis will try to be helpful in the early stages, but in general, this will make it very hard to grade, as we'd need to go through all previous submissions to verify there was no double entry.
 
+## Referral Program
+
+A key part of the program is the referral program. The purpose behind this is to incentivize an inducted or aspirational Founding Member to get more people onboarded. They way it works is:
```
Comment (blrhc) [inline README.md:72]:
```suggestion
A key part of the scheme is the referral program. The motivation behind this is to incentivize inducted or prospective Founding Members to get more people participating in the project. It works as follows:
```

File: README.md [hunk 5]
```diff
@@ -66,6 +67,17 @@ Some important considerations:
...
+1. Whenever someone [submits](/SUBMISSION-GUIDELINES.md) their scoring summary, they can list a referrer. This referrer must be identified by their `memberId` and `memberHandle`.
+2. Each time a referred user submits and gets credited a "direct score" of `n` points, the referrer gets credited a "referral score" of `0.2*n`
+3. The "total score" is simply the sum of "direct score" + "total score".
+
+Note that:
+- Although a referrer can have infinite referrals, a referred user can only have one referrer (and can not be changed).
+- The referred does not lose anything by doing this, it simply means an extra `0.2*n` is created for the referrer.
```
Comment (blrhc) [inline README.md:79]:
```suggestion
- The person referred does not lose anything by doing this, it simply means an extra `0.2*n` is generated for the referrer.
```

File: SUBMISSION-GUIDELINES.md [hunk 6]
```diff
@@ -1,50 +1,86 @@
...
+
+
+## Overview
 There are two ways to submit your activities. The primary method is to use the form on our [website](#website-form), while the other is to send your summary by [email](#email) to `founding.members (at) jsgenesis.com`.
 
 Note that in either case, make sure you include all your activities up until the end of the scoring round. When you have submitted for round `n`, you can no longer include any activity that **took place before the end of that round**. Hence, anything that was concluded in round `n-1` or `n`, will not be accepted. This rule is to prevent us awarding points for the same activity more than once.
 
 
+### Node Operators and Validators
+Many have been asking how to add their Validator or nodes to their [text files](#text-file) in order to get points. A previous example was somewhat poor in that regards, so this has been modified somewhat.
```
Comment (blrhc) [inline SUBMISSION-GUIDELINES.md:21]:
```suggestion
Many have been asking how to add their Validator or node-operating activity to their [text files](#text-file) in order to get points. A previous example was somewhat poor in that regards, so this has now been updated and improved.
```

File: technical-help/README.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+  - [Overview](#overview)
+    - [Node Operators](#node-operators)
+    - [Validators](#validators)
+<!-- TOC END -->
+
+## Overview
+This page will provide some technical help that be useful for gettin points and filling out your scoring summary. 
+
+### Node Operators
+Running a node, that is not a Validator will also earn you points, but in order for us to do so, you have to "report" to the [Joystream Telemetry](https://telemetry.joystream.org/#/Joystream) in addition to the official [Polkadot Telemetry](https://telemetry.polkadot.io). This also requires you identify yourself.
```
Comment (blrhc) [inline technical-help/README.md:17]:
```suggestion
Running a node that is not a Validator will also earn you points, but in order for us to do so, you have to "report" to the [Joystream Telemetry](https://telemetry.joystream.org/#/Joystream) in addition to the official [Polkadot Telemetry](https://telemetry.polkadot.io). This also requires you identify yourself.
```

File: technical-help/README.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+===
+
+<!-- TOC START min:1 max:3 link:true asterisk:false update:true -->
+  - [Overview](#overview)
+    - [Node Operators](#node-operators)
+    - [Validators](#validators)
+<!-- TOC END -->
+
+## Overview
+This page will provide some technical help that be useful for gettin points and filling out your scoring summary.
```
Comment (blrhc) [inline technical-help/README.md:14]:
```suggestion
This page will provide some technical help which should be useful for filling out your scoring summary and earning points. 
```

File: technical-help/README.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+After your node is reporting to our telemetry, you will be able to get some information from it [here](https://telemetry.joystream.org/tracker/). Note that this page may see some changes going forward.
+
+At the time of writing, you can submit data to the telemetry server in one of two ways. Using a [flag](#flags), or manually updating the [chain-spec file](#chain-spec-file) (note that we will update the chain spec file in the future, so this may not be necessary anymore).
+
+#### Flags
+Add the following two flags when (re-)starting your node:
+```
+--telemetry-url "wss://telemetry.joystream.org/submit 0" --telemetry-url "wss://telemetry.polkadot.io/submit/ 0"
+```
+Note that the latter (polkadot) is already on by default, but by adding just the first one (joystream) will override the settings and disable polkadots telemetry.
```
Comment (blrhc) [inline technical-help/README.md:31]:
```suggestion
Note that the latter (Polkadot) is already on by default, but by adding just the first one (Joystream) will override the settings and disable Polkadot's telemetry.
```

**Review by blrhc — APPROVED** [review]
Consider committing changes, other than that looks good!

**Pull request merged**